api/src/main/java/org/pmiops/workbench/api/DataSetController.java

```diff
@@ -128,7 +128,11 @@ private void validateDataSetCreateRequest(DataSetRequest dataSetRequest) {
       throw new BadRequestException("Missing name");
     } else if (dataSetRequest.getConceptSetIds() == null
         || (dataSetRequest.getConceptSetIds().isEmpty()
```

Comment:
I would replace this with
`CollectionUtils.isEmpty(dataSetRequest.getConceptSetIds())
        && dataSetRequest.getPrePackagedConceptSet().containsAll(ImmutableList.of(PrePackagedConceptSetEnum.NONE))`